.gitignore

```diff
@@ -17,7 +17,6 @@ node_modules
 asyncapi.json
 asyncapi.yaml
 asyncapi.yml
-/test/docs
```

Comment:
Because each file is cleaned up in the `fromTemplate` test, there is no reason to ignore it, and because one of the tests require that it's not ignored.